core/protos/o2control.proto

```diff
@@ -441,4 +444,16 @@ message SetRepoDefaultRevisionReply {
 
 message Empty {
 
+}
+
+message ListIntegratedServicesReply {
+    map<string, IntegratedServiceInfo> services = 1;
+}
+
+message IntegratedServiceInfo {
+    string name = 1;
+    bool enabled = 2;
+    string endpoint = 3;
+    string connectionState = 4;
+    string data = 5;
 }
```

Comment:
@graduta new Integrated Services API call, you should be able to use it to build mini-dashboards one for each service. For now there's only this call, support for events sent over the status channel should come soon (@miltalex ).